netutils/cjson/Makefile

```diff
@@ -54,7 +54,7 @@ $(CJSON_TARBALL):
 
 $(CJSON_UNPACKNAME): $(CJSON_TARBALL)
 	@echo "Unpacking: $(CJSON_TARBALL) -> $(CJSON_UNPACKNAME)"
-	$(Q) $(UNPACK) $(CJSON_TARBALL)
+	$(Q) $(UNPACK) $(CJSON_TARBALL) --one-top-level=$(CJSON_UNPACKNAME) --strip-components=1
```

Comment:
let's change to mv? I am afraid that unzip on other OS mayn't support --one-top-level --strip-components